exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -116,7 +117,8 @@ func constructSpanData() pdata.Traces {
 	return traces
 }
 
-func constructW3CSpanData() pdata.Traces {
+// nolint:unused
+func constructW3CSpanData() ptrace.Traces {
```

Comment:
I think for now:

```suggestion
func constructW3CSpanData() pdata.Traces {
```

Reply:
I have updated the code with this change @dmitryax to not hold disabling the test :)

Reply:
Oops. Thanks :)